msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/OSHelper.java

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.aad.msal4j;
+
+public class OSHelper {
+
+    private static String os;
+    private static boolean mac;
```

Comment:
Seems odd to have three booleans if only one of which will be true at a time. Maybe an enum with MAC, WINDOWS, and LINUX (and maybe 'UNKNOWN' if that could ever happen)? You could still have the isMac, isWindows, etc. methods, but just one field with the result of the static initializaer.

Reply:
makes sense. Updated in the latest commit.